extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -345,6 +347,45 @@ export class AmpA4A extends AMP.BaseElement {
 
     /** @private {string} */
     this.safeframeVersion_ = DEFAULT_SAFEFRAME_VERSION;
+
+    /** @private @const {RefreshManager}  */
+    this.refreshManager_ = getRefreshManagerFor(this.win);
+
+    this.refreshManager_.registerElement(this.element, refresher => {
+      console.log('Eureka!');
+      this.isRefreshing_ = true;
+      this.unlayoutCallback();
```

Comment:
Calling AMP lifecycle methods directly makes me slightly nervous; it seems likely that this might force a later refactor if there are things that we want to do only in the genuine unlayout case. Maybe this and the lifecycle methods should both call a common method? But this isn't a high-confidence recommandation and it might be fine as-is.